setup.py

```diff
@@ -77,7 +77,7 @@ def get_library_meta() -> LibraryMeta:
     elif system == "Darwin":
         name = "libMeshKernelApi.dylib"
     else:
-        if not str:
+        if not system:
```

Comment:
:-)

Reply:
Already fixed in master. See: [setup.py](https://github.com/Deltares/MeshKernelPy/blob/66de45b6ad8f31fbe17a83d133b1e537aa54042d/setup.py#L81C23-L81C23) and [meshkernel/meshkernel.py](https://github.com/Deltares/MeshKernelPy/blob/66de45b6ad8f31fbe17a83d133b1e537aa54042d/meshkernel/meshkernel.py#L92).